### DISCUSSION TITLE
Billige Antennen für DTU

### DISCUSSION THREAD

**Discussion by Zurrmaxe** [tbnobody/OpenDTU#1879] [Show and tell]
![Antennen](https://github.com/tbnobody/OpenDTU/assets/164932724/898e194d-d39e-4c51-b373-ebf74d754458)

**_Achtung : Nur für das openDTU mit CMT2300A-Module anwenden !_**

Wie ich feststellen musste, werden bei den DTU-Bausätzen oft billige Antennen aus China mitgeliefert.
Diese sind leider nicht für alle Anwendungen geeignet. Man kann sie jedoch ganz einfach öffnen und sich das Innenleben einmal anschauen.

Dazu etwas Hintergrundinformation für den Laien:

Die Antennenlänge richtet sich nach der jeweiligen Frequenz. Je höher die Frequenz, desto kürzer die Antenne.
Die Antennenlänge beträgt immer ein Vielfaches oder Teiler ( 1./2./4.) der jeweiligen Wellenlänge.
Beispiel: Bei 150 Mhz beträgt die Wellenlänge 2 Meter (300/150 = 2). Ein Lamda/Viertel strahler somit 50 cm.
Bei 2,4 GHZ sind es etwa 3,1 cm und bei 865 Mhz(CMT2300A) noch 8,6 cm.(ungefähr).

Betrachtet man nun die mitglieferten China-Antennen so stellt man fest, dass diese nur für 2400 MHZ (WLAN pp) ausgelegt sind. Beim CMT2300A funktionieren sie nicht ! (Oder nur ein paar Meter !)

Für das CMT2300A-Modul sind diese Antennen daher zu verlängern.(siehe Anleitung).

Abgleich geht wie folgt :
Wenn man openDTU benutzt, kann man in der Konsole die Feldstärke in dB ablesen.
Achtung ! Es ist ein Minus-Wert ! Je weniger, desto besser !
Die Antenne etwas länger lassen und nach Bedarf in kleinen Stücken (5mm) abschneiden bis der beste Wert gefunden wurde.
Falls man sich verschnitten hat, einfach einen neuen Draht anlöten.
Eine alte Kunststoffmine aus einem Kugelschreiber wird zur Verstärkung über den dünnen Draht geschoben und hat zufällig auch etwa die richtige Länge.

mfg.
Alfred

**Comment by dg3feh** [comment]
Moin!

> Achtung ! Es ist ein Minus-Wert ! Je weniger, desto besser !

Das ist so nicht ganz richtig. RSSI ist ein mathematisch kumulierter Wert, der ein Indikator für die Empfangsfeldstärke ist. Als EInheit ist dBm gängig, also logarithmisch bezogen auf 1mW Leistung.

Sehr guter Empfang          >  -50 dBm
Guter Empfang	          >  -70 dBm
Mittelmäßiger Empfang	  >  -80 dBm
Ausreichender Empfang	  >  -90 dBm
Schlechter Empfang	  >-105 dBm
Sehr schlechter Empfang  <-106 dBm	

Besser als -80dBm ist eigentlich easy machbar. Ich fahre auf ca. 20m von Outdoor nach Indoor -44dBm.


Beste Grüße 

Holger
